### PR TITLE
refactor exec

### PR DESCRIPTION
**Pull request by pschoffer**
## Description

To make usage of `vip dev-environment wp|add-site` more familiar to other commands and to save on some code repetion I group those together.

## Steps to Test

`vip dev-environment exec -- wp|add-site ...`

### CODE REVIEW

File: src/lib/dev-environment/dev-environment-core.js [hunk 1]
```diff
@@ -148,25 +148,14 @@ export async function runWp( slug: string, args: Array<string> ) {
...
-	await landoExec( instancePath, 'wp', args );
-}
-
-export async function runAddSite( slug: string, newSiteSlug: string, newSiteTitle: string ) {
-	debug( 'Will run a wp command on env', slug, 'for', newSiteSlug, ' - ', newSiteTitle );
-
-	const instancePath = getEnvironmentPath( slug );
-
-	debug( 'Instance path for', slug, 'is:', instancePath );
+	const commad = args.shift();
```
Comment (nejclovrencic) [inline src/lib/dev-environment/dev-environment-core.js]:
```suggestion
	const command = args.shift();
```

File: src/lib/dev-environment/dev-environment-core.js [hunk 2]
```diff
@@ -148,25 +148,14 @@ export async function runWp( slug: string, args: Array<string> ) {
...
-
-	debug( 'Instance path for', slug, 'is:', instancePath );
+	const command = args.shift();
 
-	const environmentExists = fs.existsSync( instancePath );
-
-	if ( ! environmentExists ) {
-		throw new Error( 'Environment not found.' );
+	let commandArgs = [ ...args ];
+	if ( 'add-site' === commad ) {
```
Comment (nejclovrencic) [inline src/lib/dev-environment/dev-environment-core.js]:
```suggestion
	if ( 'add-site' === command ) {
```

File: src/lib/dev-environment/dev-environment-core.js [hunk 3]
```diff
@@ -148,25 +148,14 @@ export async function runWp( slug: string, args: Array<string> ) {
...
+		commandArgs = [ ...args.map( argument => argument.replace( '--new-site-', '--' ) ) ];
 	}
 
-	const args = [ '--slug', newSiteSlug, '--title', newSiteTitle ];
-
-	await landoExec( instancePath, 'add-site', args );
+	await landoExec( instancePath, commad, commandArgs );
```
Comment (nejclovrencic) [inline src/lib/dev-environment/dev-environment-core.js]:
```suggestion
	await landoExec( instancePath, command, commandArgs );
```

File: src/lib/dev-environment/dev-environment-core.js (hunk 2 above)
Comment (pschoffer) [reply src/lib/dev-environment/dev-environment-core.js]: oh yeah also noticed that now :) Thanks. Should be fixed

**Comment by nejclovrencic** [comment]
For some reason, I can't see "dev-environment" command anymore. Not even from the currently published VIP version 🤔

**Comment by pschoffer** [comment]
@nejclovrencic  do you have proxy on? As this is still behind the isVIP flag, this will not show if you are not VIP or can't be verified to be one (in my case this is often because I forgot to turn on proxy).

**Review by nejclovrencic — APPROVED** [review]

**Pull request merged**